<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputImageData</class>
 <widget class="QDialog" name="InputImageData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Context</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>316</width>
     <height>168</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="editConversionFactor"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelConversionFactor">
        <property name="text">
         <string>Conversion Factor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelUnitOfMeasurement">
        <property name="text">
         <string>Unit of Measurement</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboUnitOfMeasurement">
        <item>
         <property name="text">
          <string>Pixel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>m</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>um</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>feed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>degree</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>radiant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>grad</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Param1Name">
        <property name="maximumSize">
         <size>
          <width>16777212</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Custom Parameter 1 Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="Param1Data"/>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Param2Name">
        <property name="text">
         <string>Custom Parameter 2 Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="Param2Data"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
